Libraries/PeriphDrivers/Include/MAX32655/wdt.h

```diff
@@ -120,6 +120,16 @@ typedef enum {
     MXC_WDT_WINDOWED = 1,
 } mxc_wdt_mode_t;
 
+/**
+ * @brief Peripheral Clock settings 
+ */
+typedef enum {
+    MXC_WDT_PCLK = 0,
+    MXC_WDT_IBRO_CLK,
+    MXC_WDT_INRO_CLK,
+    MXC_WDT_ERTCO_CLK
+} mxc_wdt_clock_t;
```

Comment:
It'd be ideal if we didn't need to create this, and could just use `mxc_sys_system_clock_t` as the second parameter to the function.

```C
/**
 * @brief       Sets clock source.
 * @param       wdt             Pointer to watchdog registers.
 * @param       clock_source    Clock source.
 */
int MXC_WDT_SetClockSource(mxc_wdt_regs_t *wdt, mxc_system_clock_t clock_source);
```

The default PCLK option is problematic though...  no action needed on this unless you have any ideas.  The less enums we have to maintain that are specific to each micro the better

Reply:
As you said PCLK option is a problem for this solution. Also, using `mxc_wdt_clock_t` enum helps to use only supported clock sources of WDT. If it's not a problem, I prefer to use `mxc_wdt_clock_t` enum solution.